security/remember_me.rst

```diff
@@ -122,7 +122,8 @@ The ``remember_me`` firewall defines the following configuration options:
 ``always_remember_me`` (default value: ``false``)
     If ``true``, the value of the ``remember_me_parameter`` is ignored and the
     "Remember Me" feature is always enabled, regardless of the desire of the
-    end user.
+    end user. If you are utilising access_control you may need to
+    add IS_AUTHENTICATED_REMEMBERED to the roles key, dependent on your setup.
```

Comment:
Since the whole article is about the remember me feature I think it would be best to not hide this information inside the description of a particular option, but add a caution node somewhere instead.

Reply:
I've also just located on this page that there is a note reference the IS_AUTHENTICATED_REMEMBERED on line 214, I still fill the information can be eluded,  perhaps adding an anchor link in the two remember me sections would help uses navigate a fuller picture. I'll update this PR as so.

Reply:
Actually, on second thought I don't think this is necessary and the PR can be closed. THank you for reviewing!